### PR TITLE
Add reflect filter and other methods

### PR DESCRIPTION
**Pull request by rodrigomologni**
### Overview

This pull request includes methods to:

- reflect a dataset object across a plane
- return the actual size of a dataset object
- copy the structure (geometry and topology) or the data attributes of a dataset object
- return the number of points, the points, the bounding box or the type of a cell

### Details

This pull request includes the following methods:

- `DataSetFilters.reflect` to reflect a dataset object across a plane
- `DataObject.actual_memory_size` to return the actual size of a dataset object
- `DataObject.copy_structure` to copy the structure (geometry and topology) of a dataset object
- `DataObject.copy_attributes` to copy the data attributes of a dataset object
- `Common.cell_n_points` to return the number of points in a cell
- `Common.cell_points` to return the points of a cell
- `Common.cell_bounds` to return the bounding box of a cell
- `Common.cell_type` to return the type of a cell

Also includes:

- tests for these methods (see `tests/test_common.py`)
- an example for the `reflect` method (see `examples/01-filter/reflect.py`)

### CODE REVIEW

File: examples/01-filter/reflect.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Reflects a mesh across a plane.
+
+"""
+
+from pyvista import examples
+
+###############################################################################
+# This example demonstrates how to reflect a mesh across a plane.
+#
+# Loads an example mesh:
```
Comment (akaszynski) [inline examples/01-filter/reflect.py:16]:
```suggestion
# Load an example mesh:
```

File: examples/01-filter/reflect.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"""
+.. _ref_reflect_example:
+
+Reflect Meshes
+~~~~~~~~~~~~~~
+
+Reflects a mesh across a plane.
```
Comment (akaszynski) [inline examples/01-filter/reflect.py:7]:
```suggestion
This example reflects a mesh across a plane.
```

File: examples/01-filter/reflect.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+from pyvista import examples
+
+###############################################################################
+# This example demonstrates how to reflect a mesh across a plane.
+#
+# Loads an example mesh:
+airplane = examples.load_airplane()
+
+###############################################################################
+# Reflects the mesh across a plane parallel to Z plane and centered in -100
```
Comment (akaszynski) [inline examples/01-filter/reflect.py:20]:
```suggestion
# Reflect the mesh across a plane parallel to Z plane and centered in -100
```

File: examples/01-filter/reflect.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# (the geometry input is copied to the output):
+airplane = airplane.reflect('z', copy=True, center=-100)
+
+###############################################################################
+# Plots the reflected mesh:
```
Comment (akaszynski) [inline examples/01-filter/reflect.py:25]:
```suggestion
# Plot the reflected mesh:
```

File: pyvista/core/common.py [hunk 5]
```diff
@@ -1035,3 +1049,70 @@ def find_closest_cell(self, point):
...
+        ----------
+        ind : int
+            Cell ID.
+
+        Returns
+        -------
+        int
+            Number of points in the cell.
+
+        """
```
Comment (akaszynski) [inline pyvista/core/common.py:1066]: Would be helpful to have an example here to demonstrate this method in action.

File: pyvista/core/common.py [hunk 6]
```diff
@@ -1035,3 +1049,70 @@ def find_closest_cell(self, point):
...
+            Cell ID.
+
+        Returns
+        -------
+        list(float)
+            The limits of the cell in the X, Y and Z directions respectivelly.
+
+        """
+        bounds = self.GetCell(ind).GetBounds()
+        return list(bounds)
```
Comment (akaszynski) [inline pyvista/core/common.py:1102]:
```suggestion
        return list(self.GetCell(ind).GetBounds())
```
Seems like we could get this on one line.

File: pyvista/core/filters.py [hunk 7]
```diff
@@ -2634,6 +2634,38 @@ def shrink(dataset, shrink_factor=1.0, progress_bar=False):
...
             return output.extract_surface()
 
+    def reflect(dataset, plane, copy=False, center=0):
+        """Reflect a dataset across a plane.
+
+        Parameters
+        ----------
+        plane : str
+            Reflection plane options: `'xmin'`, `'ymin'`, `'zmin'`, `'xmax'`, `'ymax'`, `'zmax'`,
+            `'x'`, `'y'`, or `'z'`.
```
Comment (akaszynski) [inline pyvista/core/filters.py:2644]:
```suggestion
            Reflection plane options: ``'xmin'``, ``'ymin'``,
            ``'zmin'``, ``'xmax'``, ``'ymax'``, ``'zmax'``, ``'x'``,
            ``'y'``, or ``'z'``.
```
Our docs are generated by sphinx, which follows the rst format which requires two "`" when declaring a literal.

File: pyvista/core/filters.py [hunk 8]
```diff
@@ -2634,6 +2634,38 @@ def shrink(dataset, shrink_factor=1.0, progress_bar=False):
...
+        copy : bool
+            If `True`, copy the input geometry to the output.
+        center : float
+            If the reflection plane is set to `'x'`, `'y'` or `'z'`, then this parameter is used to
+            set the position of the plane.
+
```
Comment (akaszynski) [inline pyvista/core/filters.py:2650]:
```suggestion
            If the reflection plane is set to ``'x'``, ``'y'`` or ``'z'``, then this parameter is used to
            set the position of the plane.

```

**Review by akaszynski — COMMENTED** [review]
Nice work!  I suggested changing the tense of the example along with some minor changes to the doc strings or returns.

I'd really like to see all these methods have examples.  One of the things that's most helpful with `numpy`'s documentation is that I can see how the method works in code rather than seeing the description.  Perhaps I've been staring at the screen too long, but for me I learn better through code than english :).

Not all our methods have examples, and that's something we need to work on (especially with new PRs).

File: pyvista/core/common.py (hunk 5 above)
Comment (rodrigomologni) [reply pyvista/core/common.py:1066]: Ok. I'll do it.

File: pyvista/core/common.py (hunk 6 above)
Comment (rodrigomologni) [reply pyvista/core/common.py:1102]: Yes! :sweat_smile:

File: pyvista/core/filters.py (hunk 7 above)
Comment (rodrigomologni) [reply pyvista/core/filters.py:2644]: Ok! Thanks!

**Comment by rodrigomologni** [comment]
Thank you for your changes! I will add examples as requested. :wink:

**Review by akaszynski — APPROVED** [review]
Nicely done!  Thanks for the additional examples doc strings.

**Comment by akaszynski** [comment]
@rodrigomologni, could you create a new PR?  I completely missed that the unit tests were not passing (or even being reported!).

**Comment by akaszynski** [comment]
For reference, here are the failing tests:
https://dev.azure.com/pyvista/PyVista/_build/results?buildId=4523

Be sure to run:
```
pytest -v --doctest-modules pyvista
```

We (should) verify the docstrings examples are all passing.  This way if our API changes, our examples change along with it.

**Comment by rodrigomologni** [comment]
@akaszynski I created a new PR using the same commits. See #1121, please. At worst, I change the code again.

**Comment by rodrigomologni** [comment]
**Observation:**

I ran all the tests required in Contributing before I created this PR. The following Windows error occur in `make doctest`:

image

More than one examples causes this same error. To overcome this problem, I removed all the examples, except `reflect.py`. The test was successful. Then I created this PR.

The Windows error occur same using the PyVista's master branch. I don't know whether this is a problem on my PC or Windows.

**Comment by rodrigomologni** [comment]
@akaszynski

I rewrote the changes and fixed the docstring errors using "# doctest:+SKIP"
I created a new PR again: #1122

`python -m pytest -v --doctest-modules pyvista`:

 ```python
============================= test session starts =============================
platform win32 -- Python 3.8.7, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- C:\Python38\python.exe
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('C:\\Users\\Rodrigo Mologni\\GitHub\\pyvista\\.hypothesis\\examples')
rootdir: C:\Users\Rodrigo Mologni\GitHub\pyvista, configfile: pytest.ini
plugins: hypothesis-6.0.4, cov-2.11.1, memprof-0.2.0, sphinx-0.3.1
collecting ... collected 107 items

pyvista/core/common.py::pyvista.core.common.Common.cell_bounds PASSED    [  0%]
pyvista/core/common.py::pyvista.core.common.Common.cell_n_points PASSED  [  1%]
pyvista/core/common.py::pyvista.core.common.Common.cell_points PASSED    [  2%]
pyvista/core/common.py::pyvista.core.common.Common.cell_type PASSED      [  3%]
pyvista/core/common.py::pyvista.core.common.Common.find_closest_cell PASSED [  4%]
pyvista/core/common.py::pyvista.core.common.DataObject.actual_memory_size PASSED [  5%]
pyvista/core/common.py::pyvista.core.common.DataObject.copy_attributes PASSED [  6%]
pyvista/core/common.py::pyvista.core.common.DataObject.copy_structure PASSED [  7%]
pyvista/core/composite.py::pyvista.core.composite.MultiBlock PASSED      [  8%]
pyvista/core/composite.py::pyvista.core.composite.MultiBlock.__setitem__ PASSED [  9%]
pyvista/core/filters.py::pyvista.core.filters PASSED                     [ 10%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.clip PASSED [ 11%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.clip_box PASSED [ 12%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.clip_scalar PASSED [ 13%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.compute_implicit_distance PASSED [ 14%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.probe PASSED [ 14%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.reflect PASSED [ 15%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.shrink PASSED [ 16%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.texture_map_to_sphere PASSED [ 17%]
pyvista/core/filters.py::pyvista.core.filters.DataSetFilters.threshold PASSED [ 18%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.clean PASSED [ 19%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.clip_closed_surface PASSED [ 20%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.compute_arc_length PASSED [ 21%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.compute_normals PASSED [ 22%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.decimate PASSED [ 23%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.delaunay_2d PASSED [ 24%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.extrude PASSED [ 25%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.fill_holes PASSED [ 26%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.flip_normals PASSED [ 27%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.geodesic PASSED [ 28%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.geodesic_distance PASSED [ 28%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.intersection PASSED [ 29%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.multi_ray_trace SKIPPED [ 30%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.project_points_to_plane PASSED [ 31%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.ray_trace PASSED [ 32%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.remove_points PASSED [ 33%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.ribbon PASSED [ 34%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.smooth PASSED [ 35%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.strip PASSED [ 36%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.subdivide PASSED [ 37%]
pyvista/core/filters.py::pyvista.core.filters.PolyDataFilters.tube PASSED [ 38%]
pyvista/core/filters.py::pyvista.core.filters.StructuredGridFilters.concatenate PASSED [ 39%]
pyvista/core/filters.py::pyvista.core.filters.StructuredGridFilters.extract_subset PASSED [ 40%]
pyvista/core/grid.py::pyvista.core.grid.RectilinearGrid PASSED           [ 41%]
pyvista/core/grid.py::pyvista.core.grid.UniformGrid PASSED               [ 42%]
pyvista/core/objects.py::pyvista.core.objects.Table PASSED               [ 42%]
pyvista/core/pointset.py::pyvista.core.pointset.PointSet.remove_cells PASSED [ 43%]
pyvista/core/pointset.py::pyvista.core.pointset.PolyData PASSED          [ 44%]
pyvista/core/pointset.py::pyvista.core.pointset.PolyData._from_arrays PASSED [ 45%]
pyvista/core/pointset.py::pyvista.core.pointset.StructuredGrid PASSED    [ 46%]
pyvista/core/pointset.py::pyvista.core.pointset.StructuredGrid.hide_cells PASSED [ 47%]
pyvista/core/pointset.py::pyvista.core.pointset.UnstructuredGrid PASSED  [ 48%]
pyvista/core/pointset.py::pyvista.core.pointset.UnstructuredGrid._from_arrays PASSED [ 49%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera PASSED        [ 50%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.clipping_range PASSED [ 51%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.distance PASSED [ 52%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.focal_point PASSED [ 53%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.model_transform_matrix PASSED [ 54%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.parallel_scale PASSED [ 55%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.position PASSED [ 56%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.thickness PASSED [ 57%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.up PASSED     [ 57%]
pyvista/plotting/camera.py::pyvista.plotting.camera.Camera.zoom PASSED   [ 58%]
pyvista/plotting/helpers.py::pyvista.plotting.helpers.plot_arrows PASSED [ 59%]
pyvista/plotting/itkplotter.py::pyvista.plotting.itkplotter.PlotterITK PASSED [ 60%]
pyvista/plotting/itkplotter.py::pyvista.plotting.itkplotter.PlotterITK.add_points SKIPPED [ 61%]
pyvista/plotting/plotting.py::pyvista.plotting.plotting.BasePlotter.add_arrows PASSED [ 62%]
pyvista/plotting/plotting.py::pyvista.plotting.plotting.BasePlotter.add_background_image PASSED [ 63%]
pyvista/plotting/plotting.py::pyvista.plotting.plotting.BasePlotter.add_legend PASSED [ 64%]
pyvista/plotting/plotting.py::pyvista.plotting.plotting.BasePlotter.screenshot PASSED [ 65%]
pyvista/plotting/plotting.py::pyvista.plotting.plotting.Plotter PASSED   [ 66%]
pyvista/plotting/plotting.py::pyvista.plotting.plotting.Plotter.show SKIPPED [ 67%]
pyvista/plotting/renderer.py::pyvista.plotting.renderer.Renderer.show_bounds PASSED [ 68%]
pyvista/plotting/tools.py::pyvista.plotting.tools.opacity_transfer_function PASSED [ 69%]
pyvista/utilities/cells.py::pyvista.utilities.cells.CellArray PASSED     [ 70%]
pyvista/utilities/cells.py::pyvista.utilities.cells.create_mixed_cells PASSED [ 71%]
pyvista/utilities/features.py::pyvista.utilities.features.voxelize PASSED [ 71%]
pyvista/utilities/fileio.py::pyvista.utilities.fileio.read PASSED        [ 72%]
pyvista/utilities/geometric_objects.py::pyvista.utilities.geometric_objects.CircularArc PASSED [ 73%]
pyvista/utilities/geometric_objects.py::pyvista.utilities.geometric_objects.Cylinder PASSED [ 74%]
pyvista/utilities/geometric_objects.py::pyvista.utilities.geometric_objects.Pyramid PASSED [ 75%]
pyvista/utilities/helpers.py::pyvista.utilities.helpers.line_segments_from_points PASSED [ 76%]
pyvista/utilities/helpers.py::pyvista.utilities.helpers.make_tri_mesh PASSED [ 77%]
pyvista/utilities/helpers.py::pyvista.utilities.helpers.wrap PASSED      [ 78%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricBohemianDome PASSED [ 79%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricBour PASSED [ 80%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricBoy PASSED [ 81%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricCatalanMinimal PASSED [ 82%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricConicSpiral PASSED [ 83%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricCrossCap PASSED [ 84%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricDini PASSED [ 85%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricEllipsoid PASSED [ 85%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricEnneper PASSED [ 86%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricFigure8Klein PASSED [ 87%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricHenneberg PASSED [ 88%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricKlein PASSED [ 89%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricKuen PASSED [ 90%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricMobius PASSED [ 91%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricPluckerConoid PASSED [ 92%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricPseudosphere PASSED [ 93%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricRandomHills PASSED [ 94%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricRoman PASSED [ 95%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricSuperEllipsoid PASSED [ 96%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricSuperToroid PASSED [ 97%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.ParametricTorus PASSED [ 98%]
pyvista/utilities/parametric_objects.py::pyvista.utilities.parametric_objects.Spline PASSED [ 99%]
pyvista/utilities/regression.py::pyvista.utilities.regression.compare_images PASSED [100%]

============================== warnings summary ===============================
pyvista/core/pointset.py::pyvista.core.pointset.UnstructuredGrid._from_arrays
  VTK 9 no longer accepts an offset array

-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== memory consumption estimates =========================
utilities::helpers.py::pyvista.utilities.helpers.wrap                                            - 11.4 MB
plotting::renderer.py::pyvista.plotting.renderer.Renderer.show_bounds                            - 9.0 MB
core::filters.py::pyvista.core.filters.PolyDataFilters.decimate                                  - 8.9 MB
utilities::geometric_objects.py::pyvista.utilities.geometric_objects.CircularArc                 - 6.8 MB
plotting::plotting.py::pyvista.plotting.plotting.BasePlotter.add_background_image                - 6.6 MB
core::filters.py::pyvista.core.filters.DataSetFilters.compute_implicit_distance                  - 6.2 MB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricConicSpiral     - 4.2 MB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricBohemianDome    - 2.7 MB
plotting::tools.py::pyvista.plotting.tools.opacity_transfer_function                             - 2.6 MB
core::filters.py::pyvista.core.filters.PolyDataFilters.intersection                              - 1.2 MB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricCatalanMinimal  - 980.0 KB
core::filters.py::pyvista.core.filters                                                           - 900.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricHenneberg       - 620.0 KB
core::common.py::pyvista.core.common.Common.cell_bounds                                          - 472.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricCrossCap        - 468.0 KB
core::filters.py::pyvista.core.filters.DataSetFilters.probe                                      - 392.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.compute_arc_length                        - 388.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricBoy             - 364.0 KB
core::filters.py::pyvista.core.filters.DataSetFilters.clip                                       - 300.0 KB
core::common.py::pyvista.core.common.Common.find_closest_cell                                    - 284.0 KB
utilities::features.py::pyvista.utilities.features.voxelize                                      - 284.0 KB
core::filters.py::pyvista.core.filters.DataSetFilters.clip_box                                   - 252.0 KB
core::common.py::pyvista.core.common.DataObject.copy_attributes                                  - 228.0 KB
core::pointset.py::pyvista.core.pointset.PointSet.remove_cells                                   - 156.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricSuperToroid     - 152.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricRandomHills     - 148.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricTorus           - 124.0 KB
core::composite.py::pyvista.core.composite.MultiBlock                                            - 116.0 KB
core::filters.py::pyvista.core.filters.DataSetFilters.clip_scalar                                - 104.0 KB
plotting::camera.py::pyvista.plotting.camera.Camera.model_transform_matrix                       - 104.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.compute_normals                           - 88.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricEllipsoid       - 80.0 KB
plotting::plotting.py::pyvista.plotting.plotting.BasePlotter.add_legend                          - 76.0 KB
plotting::plotting.py::pyvista.plotting.plotting.BasePlotter.screenshot                          - 76.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.strip                                     - 68.0 KB
plotting::plotting.py::pyvista.plotting.plotting.BasePlotter.add_arrows                          - 68.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricEnneper         - 68.0 KB
core::pointset.py::pyvista.core.pointset.PolyData._from_arrays                                   - 64.0 KB
plotting::camera.py::pyvista.plotting.camera.Camera.thickness                                    - 64.0 KB
plotting::helpers.py::pyvista.plotting.helpers.plot_arrows                                       - 64.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricFigure8Klein    - 64.0 KB
utilities::regression.py::pyvista.utilities.regression.compare_images                            - 64.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricRoman           - 56.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.delaunay_2d                               - 48.0 KB
core::composite.py::pyvista.core.composite.MultiBlock.__setitem__                                - 40.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.tube                                      - 40.0 KB
core::filters.py::pyvista.core.filters.StructuredGridFilters.concatenate                         - 40.0 KB
core::pointset.py::pyvista.core.pointset.PolyData                                                - 36.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.remove_points                             - 28.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.subdivide                                 - 28.0 KB
core::objects.py::pyvista.core.objects.Table                                                     - 28.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.ribbon                                    - 24.0 KB
core::grid.py::pyvista.core.grid.UniformGrid                                                     - 24.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.project_points_to_plane                   - 20.0 KB
core::pointset.py::pyvista.core.pointset.UnstructuredGrid._from_arrays                           - 20.0 KB
utilities::fileio.py::pyvista.utilities.fileio.read                                              - 20.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricKuen            - 20.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.extrude                                   - 16.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.fill_holes                                - 16.0 KB
core::grid.py::pyvista.core.grid.RectilinearGrid                                                 - 16.0 KB
core::filters.py::pyvista.core.filters.PolyDataFilters.ray_trace                                 - 12.0 KB
utilities::geometric_objects.py::pyvista.utilities.geometric_objects.Pyramid                     - 12.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricKlein           - 12.0 KB
utilities::cells.py::pyvista.utilities.cells.create_mixed_cells                                  - 8.0 KB
utilities::helpers.py::pyvista.utilities.helpers.line_segments_from_points                       - 8.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.Spline                    - 8.0 KB
core::common.py::pyvista.core.common.DataObject.copy_structure                                   - 4.0 KB
plotting::camera.py::pyvista.plotting.camera.Camera                                              - 4.0 KB
plotting::camera.py::pyvista.plotting.camera.Camera.clipping_range                               - 4.0 KB
utilities::geometric_objects.py::pyvista.utilities.geometric_objects.Cylinder                    - 4.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricBour            - 4.0 KB
utilities::parametric_objects.py::pyvista.utilities.parametric_objects.ParametricPseudosphere    - 4.0 KB
================== 104 passed, 3 skipped, 1 warning in 5.97s ==================
```

**Pull request merged**